packages/swarmplot/src/SwarmPlotCanvas.tsx

```diff
@@ -100,7 +101,7 @@ export const InnerSwarmPlotCanvas = <RawDatum,>({
         id,
         value,
         valueFormat,
-        valueScale: defaultProps.valueScale,
+        valueScale: valueScale || defaultProps.valueScale,
```

Comment:
I'd suggest to handle this the same way other defaults are managed, this can be confusing if we change this at some point (`valueScale = defaultProps.valueScale`).

Reply:
```suggestion
        valueScale,
```